pkgs/development/interpreters/spidermonkey/102.nix

```diff
@@ -0,0 +1,4 @@
+import ./common.nix {
+  version = "102.0";
```

Comment:
102.1.0 may have been released according to https://discourse.gnome.org/t/gnome-43-to-depend-on-spidermonkey-102/10658.